client/src/pages/Login.js

```diff
@@ -0,0 +1,82 @@
+import React, {useState} from 'react';
+import { bindActionCreators, compose } from 'redux';
+import { connect } from 'react-redux';
+import { styles } from './auth-styles';
+import { Paper, TextField, withStyles } from '@material-ui/core';
+import { Link, Redirect } from 'react-router-dom';
+import { login } from '../actions/userActions';
+
+const Login = ({classes, userStore, login}) => {
+    let [email, postEmail] = useState('');
+    let [pass, postPass] = useState('');
+
+    const handleChangePass = (e) => {
+        postPass(pass = e.target.value);
```

Comment:
why do we do: `postPass(pass = e.target.value)` vs. `postPass(e.target.value)`

Reply:
not sure, i'll change that